src/sidebar/helpers/version-data.js

```diff
@@ -35,15 +35,15 @@ export default class VersionData {
       }
     }
 
-    this.timestamp = new Date().toString();
+    this.version = '__VERSION__'; // replaced by versionify
```

Comment:
I just reordered that the properties appears in the same order both in the UI and code.